inject.js

```diff
@@ -321,11 +325,18 @@ chrome.extension.sendMessage({}, function(response) {
           controller.classList.toggle('vsc-hidden');
         } else if (action === 'drag') {
           handleDrag(v, controller);
+        } else if (action === 'fast') {
+          playVideoAtFastSpeed(v);
         }
       }
     });
   }
 
+  // CONTRACT: video has a way to set the playback rate.
```

Comment:
Just documenting the part of `video` that that function cares about. My convention. I'm happy to remove it.